### PR TITLE
GFORMS-824 - Enhance In expression

### PR DESCRIPTION
**Pull request by SandipKhedekar**

### CODE REVIEW

File: app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala [hunk 1]
```diff
@@ -41,7 +41,7 @@ object FormTemplateValidator {
 
   def fieldIds(sections: List[Section]) = indexedFieldIds(sections).map(_._1)
 
-  private def indexedFieldIds(sections: List[Section]): List[(FormComponentId, Int)] = indexedFields(sections).map {
+  def indexedFieldIds(sections: List[Section]): List[(FormComponentId, Int)] = indexedFields(sections).map {
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala:44]: Why removing `private`?

File: app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala [hunk 2]
```diff
@@ -268,7 +268,7 @@ object FormTemplateValidator {
       case Success(_)       => ""
     }
 
-  private def extractFcIds(expr: Expr): List[FormComponentId] = expr match {
+  def extractFcIds(expr: Expr): List[FormComponentId] = expr match {
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala:271]: Why removing `private`?

File: app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala (hunk 1 above)
Comment (SandipKhedekar) [reply app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala]: Done... Please check

File: app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala (hunk 2 above)
Comment (SandipKhedekar) [reply app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala]: Done... Please check

**Pull request merged**